modules/ROOT/pages/apikit-4-for-odata.adoc

```diff
@@ -0,0 +1,36 @@
+= APIkit for OData
+
+The APIkit OData Extension supports OData (Open Data Protocol) to create HTTP-based data services.
+
+The APIkit OData Extension builds the services from the entity data model (EDM), which OData services use to formalize their resource descriptions. You can write the EDM in RAML using EDM-compatible data types to represent API entities.
+
+The APIkit OData extension generates Mule flows to manage OData requests. You can use the APIkit OData extension to, for example, expose a legacy API as an OData API, orchestrate data before exposing it in Salesforce, or create a bidirectional OData API for Oracle, DB2, or Azure DB.
+
+The APIkit OData extension supports OData versions 2.0 and 4.0 and is compatible with the following software:
```

Comment:
Part of this intro that was extracted from OData 2 extension docs does not apply to APIKit module for OData 4.